advanced/am-pattern-2/templates/am/pub-store-tm/instance-2/wso2am-pattern-2-instance-1-am-volume-claims.yaml

```diff
@@ -23,14 +23,7 @@ spec:
   resources:
```

Comment:
@Chathuru the filename should change to `wso2am-pattern-2-instance-2-am-volume-claims.yaml`.

Reply:
Fix https://github.com/wso2/kubernetes-apim/pull/365/commits/3588e9f5044c3947741ef019ecda0ee150daad18